lib/spandex_datadog/api_server.ex

```diff
@@ -106,8 +106,11 @@ defmodule SpandexDatadog.ApiServer do
   """
   @spec send_trace(Trace.t(), Keyword.t()) :: :ok
   def send_trace(%Trace{} = trace, opts \\ []) do
-    timeout = Keyword.get(opts, :timeout, 30_000)
-    GenServer.call(__MODULE__, {:send_trace, trace}, timeout)
+    :telemetry.span([:spandex_datadog, :send_trace], %{trace: trace}, fn ->
+      timeout = Keyword.get(opts, :timeout, 30_000)
```

Comment:
I think in order for this option to be accepted, we'll need to add it to the Optimal schema above.
We [will probably remove Optimal as a dependency](https://github.com/spandex-project/spandex/pull/124) because it has some performance issues, but that'll be a separate PR.

Reply:
Oh wait, I misread that - this is the `:timeout` option to `APIServer.send_trace` and it was already there before. Never mind! 😅